.github/workflows/ci.yaml

```diff
@@ -209,31 +209,46 @@ jobs:
           echo "::set-output name=deployment-name::$DEPLOYMENT_NAME"
           echo "::set-output name=ref-dashed::$REF_DASHED"
 
+          if [ "${{ matrix.environment }}" = "production" ]; then
+            SANDBOX_ID="default"
+          fi
+
+          if [ "${{ matrix.environment }}" = "staging" ]; then
+            SANDBOX_ID="STAGING_SANDBOX_ID"
+          fi
+
+          echo "::set-output name=sandbox-id::$SANDBOX_ID"
```

Comment:
The idea is to change the names of the namespaces/deployments from `worker-refs-heads-master` to `worker-sandboxId`, which is `default` by default and a hash for a deployment created with the script in the future.